src/Processors/Formats/Impl/CHColumnToArrowColumn.h

```diff
@@ -14,15 +14,33 @@ namespace DB
 class CHColumnToArrowColumn
 {
 public:
-    CHColumnToArrowColumn(const Block & header, const std::string & format_name_, bool low_cardinality_as_dictionary_, bool output_string_as_string_, bool output_fixed_string_as_fixed_byte_array_);
+    struct Settings
```

Comment:
I don't want to pass 5 different bool flags into `CHColumnToArrowColumn` constructor, so I decided to move them to a separate struct.